package.json

```diff
@@ -28,7 +28,7 @@
         "kang": "1.3.0",
         "keep-alive-agent": "0.0.1",
         "keyapi": "git+https://github.com/TritonDataCenter/keyapi.git#bedd6fe",
-        "libmanta": "git+https://github.com/TritonDataCenter/node-libmanta.git#dd675b9",
+        "libmanta": "git+https://github.com/TritonDataCenter/node-libmanta.git#MANTA-5480",
```

Comment:
Please confirm: This needs changing once TritonDataCenter/node-libmanta#16 lands, meaning this lands AFTER the other one, right?

Reply:
That's right, to test I'm pointing to the node-libmanta branch that is required for this change. When this node-libmanta lands on master I'll point to the correct one.